objcgen/objcgenerator.cs

```diff
@@ -151,15 +151,17 @@ protected override void Generate (Type t)
 			var native_name = GetTypeName (t);
 			headers.WriteLine ();
 			headers.WriteLine ($"// {t.AssemblyQualifiedName}");
-			headers.WriteLine ($"@interface {native_name} : {GetTypeName (t.BaseType)}");
```

Comment:
ok 😢 makes me wonder if we should support 32bits macOS... 
seriously maybe the headers could be (later) made more modern for iOS/tvOS/watchOS